example.py

```diff
@@ -3,46 +3,52 @@
 Test psycopg with CockroachDB.
 """
 
-import time
-import random
 import logging
 import os
+import random
+import time
+import uuid
 from argparse import ArgumentParser, RawTextHelpFormatter
 
 import psycopg2
 from psycopg2.errors import SerializationFailure
+import psycopg2.extras
 
 
 def create_accounts(conn):
-    with conn.cursor() as cur:
+    psycopg2.extras.register_uuid()
+    with conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor) as cur:
         cur.execute(
-            "CREATE TABLE IF NOT EXISTS accounts (id INT PRIMARY KEY, balance INT)"
+            "CREATE TABLE IF NOT EXISTS accounts (id UUID PRIMARY KEY, balance INT)"
         )
         cur.execute(
-            "UPSERT INTO accounts (id, balance) VALUES (1, 1000), (2, 250)")
+            "UPSERT INTO accounts (id, balance) VALUES (%s, 1000), (%s, 250)", (uuid.uuid4(), uuid.uuid4()))
```

Comment:
Out of curiosity, has CRDB a `gen_random_uuid()` function?

Reply:
Yes, it does, and I considered using it here. But the other simple CRUD examples all are generating the UUIDs client-side, so I'm following their lead. E.g. https://github.com/cockroachlabs/example-app-python-sqlalchemy/blob/master/main.py#L29